app/src/main/java/com/github/dodobest/upbitapi/DataFormatHandler.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.dodobest.upbitapi
+
+import java.text.DecimalFormat
+
+class DataFormatHandler {
```

Comment:
```suggestion
object DataFormatHandler {
```
생성자로 무언가를 캡슐화하지 않는다면 object를 활용해보세요

Reply:
[3887f1d](https://github.com/DoTheBestMayB/UpbitAPI/pull/31/commits/3887f1de22e25859a431e1942f7ccf583731f97d)

object 클래스는 싱글턴이고, 싱글턴 객체는 app이 실행될 때 생성되므로 object 클래스가 많으면 초기 앱 화면이 나오는 시간이 느려진다는 단점이 있나요?

Reply:
싱글턴 객체는 앱이 실행될 때 한번만 객체를 생성하고, 그 이후에는 생성하지 않고 이전에 생성된 객체를 활용합니다.
싱글턴을 사용한다고 해서 느려지지는 않습니다만, 싱글턴은 화면 생명주기아 아니라 어플 생명주기를 따라가기 때문에 앱이 종료될 때까지 메모리에서 해제되지 않습니다.
요즘은 하드웨어 스펙이 너무 좋기도 하고, 객체가 무겁지 않다보니 성능에 영향은 거의 없다고 보시면 됩니다.